### PR TITLE
More API methods

### PR DESCRIPTION
**Pull request by Fernthedev**

### CODE REVIEW

File: shared/modloader.h [hunk 1]
```diff
@@ -60,6 +74,14 @@ MODLOADER_EXPORT extern JavaVM* modloader_jvm;
 MODLOADER_EXPORT extern void* modloader_libil2cpp_handle;
 /// @brief The captured dlopen-d libunity.so handle
 MODLOADER_EXPORT extern void* modloader_unity_handle;
+/// @brief True if libs have been dlopened
+MODLOADER_EXPORT extern bool libs_loaded;
```
Comment (RedBrumbler) [inline shared/modloader.h]: I think these should be renamed to `thing_opened` to be more clear about what they stand for, also the `early_mods_loaded` and `late_mods_loaded` are not actually what their comments describe them as. They are instead more similiar to libs, set to true when opened, not when load was called for all of them

**Comment by RedBrumbler** [comment]
the rest lgtm although the require mod call might be weird with loading a mod twice? not sure about that one though as I am unsure how already opened libs are handled by mods. maybe finding the right mod in the appropriate vector would be better?

**Comment by Fernthedev** [comment]
The mods' functions already check if they've been called already, thereby avoiding the problem.

**Comment by RedBrumbler** [comment]
wouldn't it be making a new instance of LoadedMod and thus one without those booleans set?

**Comment by Fernthedev** [comment]
Given that the copy constructor is explicitly deleted, no.

**Review by RedBrumbler — APPROVED** [review]

**Pull request merged**